### PR TITLE
Add aria-pressed for Button, so that it can be toggle button

### PR DESCRIPTION
**Pull request by yjlo**
Added a `ariaPressed` prop to button, need it for Webex web client accessibility
## Related Issue

https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-122334

## Screenshots:
**Before** (If applicable):

**After**:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Other... Please describe: aria accessibility

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: react/src/lib/Button/index.js [hunk 1]
```diff
@@ -266,6 +270,7 @@ Button.defaultProps = {
   active: false,
   ariaLabel: '',
   ariaLabelledBy: '',
+  ariaPressed: undefined,
```
Comment (andrewholsted) [inline react/src/lib/Button/index.js]: Should this really be undefined?

Comment (yjlo) [reply react/src/lib/Button/index.js]: For a non-toggle button it shouldn't have ariaPressed, not even false

Comment (andrewholsted) [reply react/src/lib/Button/index.js]: I was referring to an empty string vs undefined

Comment (yjlo) [reply react/src/lib/Button/index.js]: Okay I can change that real quick

**Pull request merged**